openfisca_us/variables/irs/inputs.py

```diff
@@ -831,6 +841,16 @@ class e24515(Variable):
     documentation = """Sch D: Un-Recaptured Section 1250 Gain"""
 
 
+class filer_e24515(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = u"Section 1250 Gain (un-recaptured) for the tax unit (excluding dependents)"
+    definition_period = YEAR
```

Comment:
should this and others also have `unit = "currency_USD"`?

Reply:
Yep, added in the rest of the changed variables